test/mocha-junit-reporter-spec.js

```diff
@@ -393,4 +394,70 @@ describe('mocha-junit-reporter', function() {
       expect(testCase.testcase[0]._attr.classname).to.equal(mockedTestCase.fullTitle());
     });
   });
+
+  describe('XML format', function () {
+    var suites = [
+      {testsuite:
+        {title: '', root: true, suites: [2], tests: [0]}
+      },
+      {testsuite:
+        {title: 'Inner Suite', suites: [1], tests: [1]}, pass: [
+          {title: 'test', fullTitle: 'Inner Suite test'}
+        ]
+      },
+      {testsuite:
+        {title: 'Another Suite', suites: [1], tests: [1]}, fail: [
+          {title: 'fail test', fullTitle: 'Another Suite fail test', error: new Error('failed test')}
+        ]
+      }
+    ];
+
+    it('generates Jenkins compatible XML when in jenkinsMode', function() {
+      var reporter = configureReporter({jenkinsMode: true }, suites);
```

Comment:
This is your only reference to `jenkinsMode` did you forget to push a commit?

Reply:
I submitted another pull request for jenkinsMode https://github.com/michaelleeallen/mocha-junit-reporter/pull/75 I just thought it make sense to add the schema test here since the default schema is compatible with jenkins schema.